mysql/datadog_checks/mysql/queries.py

```diff
@@ -17,6 +17,10 @@
 FROM     information_schema.tables
 GROUP BY table_schema"""
 
+SQL_QUERY_TABLE_ROWS_STATS = """\
+SELECT   CONCAT(table_schema,'.',table_name) as table_name, rows_read, rows_changed
+FROM     information_schema.table_statistics"""
```

Comment:
it looks like this table is only for mariadb? https://mariadb.com/kb/en/information-schema-table_statistics-table/

the analog in mysql appears to be this https://dev.mysql.com/doc/refman/5.7/en/information-schema-tables-table.html although I'm not sure if `TABLE_ROWS` is equivalent to `ROWS_READ`, for example. 

This should be made clear in the config that this feature only works for mariadb, and you should also check the mysql flavor == 'mariadb' in addition to the table size metrics flag being turned on before running the query (or make this work for both mariadb and mysql?)

Reply:
Hello @jmeunier28, table_statistics are created and managed by the "User Statistics" plugin, it is available on [MariaDB](https://mariadb.com/kb/en/user-statistics/) and [Percona MySQL](https://www.percona.com/doc/percona-server/8.0/diagnostics/user_stats.html), I don't know if it exists on Oracle MysQL but it is not a MariaDB feature. 

In the Prometheus exporter for MySQL they feature flap the metrics tablestats, schemastats, userstats and explicitly mention that the plugin should be enabled to have those metrics (`If running with userstat=1, set to true to collect user statistics.`), I guess it is a good approach, @aymeric-ledizes can you implement this?  https://github.com/prometheus/mysqld_exporter/blob/main/README.md?plain=1#L73-L75

Reply:
It's been a while this PR is open but now done with `userstat` check :)